modules/programs/neovim.nix

```
@@ -51,8 +51,8 @@ let
         (map (x: if x ? plugin && x.optional == true then x.plugin else null)
           cfg.plugins);
     };
-    customRC = cfg.extraConfig
-      + pkgs.lib.concatMapStrings pluginConfig cfg.plugins;
+    customRC = pkgs.lib.concatMapStrings pluginConfig cfg.plugins
+      + cfg.extraConfig;
```

```suggestion
    customRC = concatMapStrings pluginConfig cfg.plugins
      + cfg.extraConfig;
```

So extraConfig is appended now, is that right?
I thought this was already the case when I looked at my own configuration file.

---

me too but it wasn't. It may break some configs but I think appending extraConfig is better to let user override plugin specific config. It will be in line with the nixpkgs wrapper that will append extraConfig after the plugin config. I can remove it if you feel it's not the good time.

---

broken for broken at least it will minimize future breaking changes. Merging.